src/app/view-modes/collation/collation.component.ts

```diff
@@ -25,6 +26,24 @@ export class CollationComponent implements OnInit, OnDestroy {
     map(({ page }) => page.id),
   );
 
+  public get addWitnessBtn(): { label: string, additionalClasses: string, title: string, icon: EvtIconInfo } {
```

Comment:
Weird getter name.

Reply:
Could `addWitnessBtnLabel` be better?

Reply:
I think the name addWitnessBtn isn’t so bad

Reply:
What about removing `add` and leave just `witnessBtn`?